MQTT-C_Client/README.md

```diff
@@ -0,0 +1,82 @@
+# MQTT project
+
+The goal of this project is to show how to add MQTT support to an Azure Sphere project, in this case using the [Eclipse Mosquitto MQTT Broker](https://mosquitto.org/) and a PC-based Python application.
+
+## Contents
+
+| Folder\file | Description |
+|-------------|-------------|
+| `src\HighLevelApp`       | Azure Sphere Sample App source code |
+| `src\HighLevelApp\Certs`       | placeholder folder for MQTT certs |
+| `src\PyMqttHost`       | Python app that subscribes and publishes messages to a device.  |
+| `README.md` | This README file. |
+| `LICENSE.txt`   | The license for the project. |
+
+## Prerequisites & Setup
+
+- An Azure Sphere-based device with development features (see [Get started with Azure Sphere](https://azure.microsoft.com/en-us/services/azure-sphere/get-started/) for more information).
+- Setup a development environment for Azure Sphere (see [Quickstarts to set up your Azure Sphere device](https://docs.microsoft.com/en-us/azure-sphere/install/overview) for more information).
+- Install Python 3 from [Python.org](www.python.org).
+- Install the ```paho-mqtt``` library using ```pip3 install -r requirements.txt```.
+
+Note that the Azure Sphere high-level application is configured for the 21.01 SDK release.
+
+## How to use
+
+Download the Mosquitto Certificate Authority certificate [mosquitto.org.crt (PEM format)](https://test.mosquitto.org/) and save in the HighLevelApp/Certs folder.
+
+### Generate the client certificates with openssl
+
+For Linux users, openssl is included. For Windows users either download ```openssl``` from [openssl.org](https://wiki.openssl.org/index.php/Binaries), or use [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10) (Windows Subsystem for Linux).
```

Comment:
It's not necessarily the case that a given Linux distro includes openssl in the default installation. Maybe just say you can install it using your distribution's package manager if it's not installed by default?